modules/material-management/src/types.ts

```diff
@@ -113,3 +121,38 @@ export interface ClientOptions {
 }
 
 export type Newable<T> = { new (...args: any[]): T }
+
+export interface AwsEsdkJsonWebKey {
+  crv?: string
+  d?: string
+  dp?: string
+  dq?: string
+  e?: string
+  k?: string
+  kty?: string
+  n?: string
+  p?: string
+  q?: string
+  qi?: string
+  x?: string
+  y?: string
```

Comment:
Can we add comments explaining these? The variable names are opaque to me.

Reply:
This is a copy of the `JsonWebKey` type. They are not my invention.

/* These are copies of the v12 Node.js types.
 * I copied them here to avoid exporting v12 types
 * and forcing consumers to install/use v12 in their projects.

is the relevant comment.
It is no longer 12, and now 14 as well as some DOM types.